modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -167,3 +167,22 @@ links in the deployment's service specification file to overcome this:
     labels:
       template: deploymentConfigTemplate
 ----
+
+The number of application pods that can run in a namespace is dependent on the number of services and the
+length of the service name when the environment variables are used for service discovery. `ARG_MAX` on the system 
+defines the maximum argument length for a new process and it is set to `2097152 KiB` by default. Kubelet injects 
+environment variables in to each pod scheduled to run in the namespace including:
+
+```
+<SERVICE_NAME>_SERVICE_HOST=<IP>
```

Comment:
@chaitanyaenr Sorry if I wasn't clear. I mean formatting this section like:

    name space including:
    
    * `<SERVICE_NAME>_SERVICE_HOST=<IP>`
    * `<SERVICE_NAME>_SERVICE_PORT=<PORT>`
    * `<SERVICE_NAME>_PORT=tcp://<IP>:<PORT>`
    * `<SERVICE_NAME>_PORT_<PORT>_TCP=tcp://<IP>:<PORT>`
    * `<SERVICE_NAME>_PORT_<PORT>_TCP_PROTO=tcp`
    * `<SERVICE_NAME>_PORT_<PORT>_TCP_PORT=<PORT>`
    * `<SERVICE_NAME>_PORT_<PORT>_TCP_ADDR=<ADDR>`